libs/utils/wa_results_collector.py

```diff
@@ -972,7 +972,7 @@ def find_comparisons(self, base_id=None, by='kernel'):
                         # Find a p-value which hopefully represents the
                         # (complement of the) certainty that any difference in
                         # the mean represents something real.
-                        pvalue =  ttest_ind(group_results, base_results, equal_var=False).pvalue
+                        tmpstat, pvalue = ttest_ind(group_results, base_results, equal_var=False)
```

Comment:
I think this might be due to discrepancies between the scipy version we use and the one used in the VM. Are you using an older distro ?

Otherwise, the change is fine with me, with a tiny nitpick: since `tmpstat` is not used, you can just write that line as:
``` python
_, pvalue = ttest_ind(...)
```

Reply:
I'm using the vagrant image:
Linux vagrant-ubuntu-trusty-64 3.13.0-135-generic #184-Ubuntu SMP Wed Oct 18 11:55:51 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux

Reply:
I think some people have been running Trusty without issues, but then maybe they bypassed the package manager to get a newer version of scipy (seems that the return fields were renamed between scipy 0.15.1 and scipy 0.16.0).

In any case, the change still works on newer versions so we can go with it

Reply:
@valschneider - I think this means that the review is OK now?